<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>583</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>40</width>
    <height>369</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1105</width>
    <height>676</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Co-Text</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>992</width>
     <height>22</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active/>
     <inactive/>
     <disabled/>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionPrintPDF"/>
    <addaction name="actionShare_Uri"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="actionFind"/>
    <addaction name="actionReplace"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionLogout"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuUser"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(60, 202, 33);</string>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>42</width>
     <height>42</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogin"/>
   <addaction name="actionOpen"/>
   <addaction name="actionNew"/>
   <addaction name="actionShare_Uri"/>
   <addaction name="actionExit"/>
   <addaction name="actionSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>mainToolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFont"/>
   <addaction name="actionTextColor"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
  </widget>
  <action name="actionNew">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/icons/noun_new document_2429385.svg</normaloff>:/imgs/icons/noun_new document_2429385.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/icons/noun_Folder_2047792.svg</normaloff>
     <normalon>:/imgs/icons/noun_Folder_2047792.svg</normalon>:/imgs/icons/noun_Folder_2047792.svg</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normalon>:/imgs/icons/noun_Download_2383249.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/icons/noun_Save_1435429.png</normaloff>:/imgs/icons/noun_Save_1435429.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPrintPDF">
   <property name="text">
    <string>Export as PDF...</string>
   </property>
   <property name="toolTip">
    <string>ExportPDF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/icons/noun_Exit_1558251.svg</normaloff>:/imgs/icons/noun_Exit_1558251.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normalon>:/imgs/icons/noun_Undo_1246691.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/icons/noun_copy_2080193.png</normaloff>:/imgs/icons/noun_copy_2080193.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/icons/noun_paste_2302624.png</normaloff>:/imgs/icons/noun_paste_2302624.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normalon>:/imgs/icons/noun_Cut File_3118760.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset>
     <normalon>:/imgs/icons/noun_file find_556585.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset>
     <normalon>:/imgs/icons/noun_replace_1913722.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Replace</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/icons/noun_Redo_124669.png</normaloff>:/imgs/icons/noun_Redo_124669.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/icons/noun_user login_178831.svg</normaloff>
     <normalon>:/imgs/icons/noun_user login_178831.svg</normalon>
     <activeoff>:/imgs/icons/noun_user login_178831.svg</activeoff>
     <activeon>:/imgs/icons/noun_user login_178831_white.svg</activeon>
     <selectedoff>:/imgs/icons/noun_user login_178831.svg</selectedoff>
     <selectedon>:/imgs/icons/noun_user login_178831_white.svg</selectedon>:/imgs/icons/noun_user login_178831.svg</iconset>
   </property>
   <property name="text">
    <string>Login...</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionShare_Uri">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/icons/noun_url_887536.svg</normaloff>:/imgs/icons/noun_url_887536.svg</iconset>
   </property>
   <property name="text">
    <string>Share Uri</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/icons/noun_Settings_2324598.svg</normaloff>:/imgs/icons/noun_Settings_2324598.svg</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/icons/noun_font_2669552.svg</normaloff>:/imgs/icons/noun_font_2669552.svg</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/icons/noun_italics_1076852.svg</normaloff>:/imgs/icons/noun_italics_1076852.svg</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="toolTip">
    <string>Italic font</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/icons/noun_bold_1076853.svg</normaloff>:/imgs/icons/noun_bold_1076853.svg</iconset>
   </property>
   <property name="text">
    <string>actionBold</string>
   </property>
   <property name="toolTip">
    <string>Bold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/icons/noun_underline_1384032.svg</normaloff>:/imgs/icons/noun_underline_1384032.svg</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
   <property name="toolTip">
    <string>Underline</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionTextColor">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/icons/noun_color_3170276.svg</normaloff>:/imgs/icons/noun_color_3170276.svg</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
